package.json

```diff
@@ -1,34 +1,36 @@
 {
   "name": "@popsure/public-models",
-  "version": "1.0.0",
+  "version": "1.0.1-alpha.4",
   "author": "Vincent Audoire <vincent@getpopsure.com>",
   "license": "MIT",
   "private": false,
   "description": "Feather insurance models interface and utilities",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js",
+      "types": "./dist/index.d.ts"
+    }
+  },
   "repository": "https://github.com/getPopsure/models",
   "scripts": {
-    "build": "rm -rf dist && rollup -c",
+    "build": "rm -rf dist && tsup src/index.ts --format cjs,esm --dts --sourcemap",
     "release": "yarn build && yarn publish --access public",
     "test": "jest"
   },
   "devDependencies": {
     "@babel/core": "^7.9.0",
     "@babel/preset-env": "^7.9.5",
     "@babel/preset-typescript": "^7.9.0",
-    "@rollup/plugin-json": "^4.1.0",
-    "@rollup/plugin-typescript": "^4.1.1",
     "@types/jest": "^25.2.1",
     "babel-jest": "^25.3.0",
     "jest": "^27.5.1",
     "prettier": "2.0.4",
```

Comment:
btw prettier is not used right now

Reply:
But don't we need to add it in order for devs to have it in their local dev environment? Otherwise we need to ask people to add prettier globally.